app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,9 @@
 class PostsController < ApplicationController
+    def index
+      @posts = Post.all
+      render :index
```

Comment:
I don't think you will need this line for it to default to the index template. That being said a case could be made to have it explicit in the code. What's important is that you and your team are consistent with these things.

Reply:
Do you mean the render line?